evap/staff/tests/test_forms.py

```diff
@@ -546,7 +547,10 @@ def test_contributor_evaluation_form_date_validation(self):
         evaluation.general_contribution.questionnaires.set([mommy.make(Questionnaire)])
 
         # contributors: start date must be in the future
-        self.helper_date_validation(ContributorEvaluationForm, "1999-01-01", "2099-01-01", False)
+        # since issue #1312, a past vote_start_datetime is automatically set to datetime.now() (rounded up)
+        # -> test wrong, instead test whether form is still valid
+        # self.helper_date_validation(ContributorEvaluationForm, "1999-01-01", "2099-01-01", False)
```

Comment:
Code comments are not the place to keep the whole history, we have git for that. I vote to remove the original comment and just replace it by
"since issue #1312, a past vote_start_datetime is automatically set to datetime.now() (ceiled). Thus, this form should be valid"

Reply:
(Especially do we not need or want commented code)